src/strptime.h

```diff
@@ -0,0 +1,50 @@
+
+/*
+ * Polyfill for strptime function
```

Comment:
Are there any other alternatives to this polyfill, e.g., https://docs.microsoft.com/en-us/answers/questions/572611/strptime-alternative-in-mingw-vs2019.html ?

Reply:
Perhaps we can call `std::get_time` instead?  This only exists on C++11 and s3fs only requires C++03 but it seems better for the win32 port to use this mode instead of importing this code.

Reply:
Replaced the polyfill with the code at d5f1101 .